dbldatagen/data_generator.py

```diff
@@ -1604,3 +1617,51 @@ def scriptMerge(self, tgtName=None, srcName=None, updateExpr=None, delExpr=None,
             result = HtmlUtils.formatCodeAsHtml(results)
 
         return result
+
+    @staticmethod
+    def fromDict(options):
+        """ Creates a data generator from a dictionary of options.
+
+            :param options: Dictionary with data generator options (e.g. "name", "rows")
+            :return: A data generator with the specified options
+        """
+        return DataGenerator(**options)
+
+    @staticmethod
+    def fromFile(path):
```

Comment:
Dont add fromFile as method as `open` does not support reading file from a Databricks workspace or dbfs